applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_solver.py

```diff
@@ -113,7 +113,11 @@ def GetDefaultSettings(cls):
             "reform_dofs_at_each_step": false,
             "line_search": false,
             "compute_reactions": true,
-            "block_builder": true,
+            "block_builder" : true,
+            "builder_and_solver_settings" : {
+                "diagonal_values_for_dirichlet_dofs" : "use_max_diagonal",
```

Comment:
is this a behavior change or not?
I am lost

Reply:
Yes, it is, but @RiccardoRossi suggested that

Reply:
ok

Reply:
I think that "use_max_diagonal" is better, and that we should eventually use it as default. not sure if it will break any test... (for example using max_diagonal should guarantee, by the gershgorin theorem, that the lowest eigenvalues are not affected and hence enable using the block builder and solver with the eigenvalue solution)